app/models/instrument.rb

```diff
@@ -31,6 +31,13 @@ class Instrument < Product
 
   after_create :set_default_pricing
 
+  # Scopes
+  # --------
+  scope :reservation_only, lambda {
```

Comment:
Probably should have mentioned this when I saw it earlier. We've been preferring class-method style scopes over `scope`.